packages/jest-matchers/src/index.js

```diff
@@ -20,6 +20,13 @@ import type {
 } from 'types/Matchers';
 
 const utils = require('jest-matcher-utils');
+const extractExpectedAssertionsErrors = require('./extractExpectedAssertionsErrors');
+const {
+  getMatchers,
+  getState,
+  setMatchers,
+  setState,
```

Comment:
it's still namespaced within the matchers package though.
and with this naming the other two functions will become `getMatchersMatchers` then :D